mlrun/model_monitoring/application.py

```diff
@@ -84,16 +84,16 @@ class ModelMonitoringApplicationBase(StepToDict, ABC):
         class MyApp(ApplicationBase):
             def do_tracking(
                 self,
-                sample_df_stats: pd.DataFrame,
-                feature_stats: pd.DataFrame,
+                sample_df_stats: mlrun.common.model_monitoring.helpers.FeatureStats,
+                feature_stats: mlrun.common.model_monitoring.helpers.FeatureStats,
```

Comment:
Make sure that you didn't break any backwards compatibility + how does it affect the user app code? when he defines a custom application should he look up for this pre-defined type FeatureStats?

Reply:
> Make sure that you didn't break any backwards compatibility

We change the API here, but we plan to address the BC issue in `do_tracking` as a part of a more general API improvement: [ML-5494](https://iguazio.atlassian.net/browse/ML-5494).
This should be at least documented in a release note.

> how does it affect the user app code? when he defines a custom application should he look up for this pre-defined type FeatureStats?

Users who want the previous data frames can use `self.dict_to_histogram`.

Reply:
We agreed to address the BC issue as a part of [ML-5494](https://iguazio.atlassian.net/browse/ML-5494), which is directly related.